type.proto

```diff
@@ -546,6 +546,12 @@ message LocationContext {
     required int32 access_duration = 2;
 }
 
+enum DistanceStatus {
+    Reachable = 0;
```

Comment:
I would more go for `reached` and `unreached` (camelcase, lowercase for the major part of enums it seems, so `unknown` also)